stock_request_analytic/tests/test_stock_request_analytic.py

```diff
@@ -42,8 +42,9 @@ def setUp(self):
         })
         self.pizza.route_ids = [(6, 0, self.demand_route.ids)]
 
-    def prepare_order_request_analytic(self, aa, company):
+    def prepare_order_request_analytic(self, aa, company, ats=None):
```

Comment:
Significant argument names

```suggestion
    def prepare_order_request_analytic(self, aa, company, analytic_tags=None):
```